site/docs/v1/tech/apis/authentication.adoc

```diff
@@ -158,6 +158,8 @@ Cookie: access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE0ODUxNDA5OD
 
 When an API that is marked with a green unlocked icon such as &nbsp;icon:unlock-alt[role=green, title="No authentication required",type=fas]&nbsp; it means that you are not required to provide an `Authorization` header as part of the request. The API is either designed to be publicly accessible or the request may take a parameter that is in itself secure.
 
+Some APIs may be authenticated by the source IP address of the request. For example, if `fusionauth-app.local-metrics.enabled` is set to `true`,  `/api/prometheus/metrics` and `/api/status` will accept requests from `localhost`.
```

Comment:
Would it make sense to have a separate section for authentication via source IP?

Reply:
I added the new section and updated the wording, and think it looks better this way.